resources/views/common/head.html.twig

```diff
@@ -91,6 +91,12 @@
         integrity="sha256-VazP97ZCwtekAsvgPBSUwPFKdrwD3unUfSGVYrahUqU="
         crossorigin="anonymous"></script>
 
+{# Clone JS #}
+
+    <script
+        src="https://cdn.jsdelivr.net/npm/clone-framework@2.2.1/dist/js/clone.min.js">
```

Comment:
New sources added to the <head> will need a SHA integrity code / be added to the CSP header.